### DISCUSSION TITLE
Graphql-helix v1.11 incompatibility

### DISCUSSION THREAD

**Discussion by kefniark** [enisdenjo/graphql-sse#12] [Q&A] [answered]
## Description
In their last version, `graphql-helix` decided to change how they handle subscriptions over SSE.
* Release Note: https://github.com/contra/graphql-helix/releases/tag/graphql-helix%401.11.0
* Related Issue: https://github.com/contra/graphql-helix/issues/224

Now, they don't support SSE over POST anymore, and it breaks the way `graphql-sse` client works with their recommended [subscription over SSE](https://graphql-helix.vercel.app/docs/recipes/subscriptions-over-sse)
Do you see any alternative to make it work with this library, and more simply I would like to have your POV on that.

**Comment by enisdenjo** [comment, marked as answer]
Duplicate of https://github.com/enisdenjo/graphql-sse/discussions/10.

`graphql-helix` wasn't ever compatible with `graphql-sse`. Also, see the comment in the issue you referenced https://github.com/contra/graphql-helix/issues/224#issuecomment-999217349.
